todo-app/src/list.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ListItem from './ListItem'
+import './App.css';
+
+ export default class List extends React.Component {
+
+render() {
+
+    return (
+      <div className='todo-items'>
+        <ListItem description={this.props.description}
```

Comment:
If you agree with my previous comment about `List` and `ListItem`, and assuming `List` component would receive an array of `todos` as a prop, then here you would write something like

```javascript
const todos = this.props.todos;
{todos.map((item, index) => {
  return (
    <ListItem
      key={index}
      index={index}
      // the rest of the properties here
    />
  );
}}
```